base/bucket_gocb.go

```diff
@@ -2380,3 +2438,47 @@ func (bucket *CouchbaseBucketGoCB) releaseViewOp() {
 	<-bucket.viewOps
 	gocbExpvars.Add("ViewOps", -1)
 }
+
+func GoCBBucketItemCount(bucket *gocb.Bucket, spec BucketSpec, user, pass string) (itemCount int, err error) {
+
+	reqUri := fmt.Sprintf("%s/pools/default/buckets/%s", spec.Server, spec.BucketName)
```

Comment:
Does this work if spec.Server is 'couchbase://'?  

I think it would be preferable to use goCBClient.MgmtEps to get this endpoint - it feels more reliable.

Reply:
Fixed in 2cb91a569f34292d8a3c3d03b076585f343fd7da